ccmlib/cluster.py

```diff
@@ -688,3 +688,26 @@ def timed_grep_nodes_for_patterns(self, versions_to_patterns, timeout_seconds, f
                     ret = namedtuple('Node_Log_Matching', 'node matchings')
                     return ret(node=node, matchings=matchings)
             time.sleep(1)
+
+    def wait_for_any_log(self, nodes, pattern, timeout, filename='system.log'):
```

Comment:
What you'll want to do here, rather than essentially duplicating the function, is to wrap the common.py function. Additionally, you don't want to take a list of nodes, but just use `cluster.nodelist()`.

So, in pseudocode, it would roughly look like:
```
def wait_for_any_log(self, pattern, timeout, filename='system.log'):
   return common.wait_for_any_log(self.nodelist(), pattern, timeout, filename=filename
```